packages/@storefront/flux-capacitor/src/core/actions/creators.ts

```diff
@@ -1176,6 +1176,23 @@ namespace ActionCreators {
   export function startApp(): Actions.StartApp {
     return createAction(Actions.START_APP, undefined);
   }
+
+  // navigation action creators
+  /**
+   * Fires the START_REDIRECT action.
+   * @return - Action with undefined.
+   */
+  export function startRedirect(): Actions.StartRedirect {
+    return createAction(Actions.START_REDIRECT, undefined);
```

Comment:
Why are you returning `undefined` as the action's payload?

Reply:
Currently, type declarations for actions that do not include a payload use one of two values: `undefined`; or `any` (see the links below for examples). In this case, I opted to use `undefined`.

https://github.com/groupby/storefront/blob/master/packages/%40storefront/flux-capacitor/src/core/actions/index.ts#L71

https://github.com/groupby/storefront/blob/master/packages/%40storefront/flux-capacitor/src/core/actions/index.ts#L237

Reply:
Cool makes sense. Resolving this thread.